<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainTabWidget</class>
    <widget class="QWidget" name="MainTabWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>leafxy app</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QTabWidget" name="tabWidget">
                    <property name="tabPosition">
                        <enum>QTabWidget::South</enum>
                    </property>
                    <property name="tabShape">
                        <enum>QTabWidget::Triangular</enum>
                    </property>
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="wel_tab">
                        <attribute name="title">
                            <string>welcome</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="draw_tab">
                        <attribute name="title">
                            <string>drawing</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="v2d_tab">
                        <attribute name="title">
                            <string>2d view</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="v3d_tab">
                        <attribute name="title">
                            <string>3d view</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="img_tab">
                        <attribute name="title">
                            <string>image view</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="cam_tab">
                        <attribute name="title">
                            <string>camera view</string>
                        </attribute>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
